fiasko_bro/code_validator.py

```diff
@@ -9,6 +10,24 @@ def validate_repo(path_to_repo, path_to_original_repo=None):
     return code_validator.validate(path_to_repo, path_to_original_repo)
 
 
+def py_files_count(path_to_repo):
+    counter = 0
+    for _, _, files in os.walk(path_to_repo):
+        for file in files:
+            if file.endswith('.py'):
+                counter += 1
+    return counter
+
+
+def is_repos_too_large(path_to_repo, path_to_original_repo, max_num):
```

Comment:
`max_num` is a bad name (max num of what?). I suggest `max_py_files_count`. Don't worry too much if a function definition exceeds 80-char limit.